### PR TITLE
Shakemap map

### LINKED ISSUE #925: Map for Shakemap module

### LINKED ISSUE #926: Shakemap map overlays pipe

### PR DESCRIPTION
**Pull request by dslosky-usgs**
Closes #925 closes #926 

Initial shakemap map working with InteractiveMapOverlaysPipe and queryParams to display the correct layers.

### CODE REVIEW

File: src/app/shakemap/map/map.component.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<ng-container *ngIf="eventService?.event$ | async; let event">
+  <ng-container *ngIf="event | getProduct:'shakemap' as shakemap">
+    <ng-container *ngIf="activatedRoute.queryParamMap | async; let params">
```
Comment (jmfee-usgs) [inline src/app/shakemap/map/map.component.html]: Shakemap doesn't need to do this.

you should only have:
```
<ng-container *ngIf="eventService.product$ | async; let product">
  ...
  <shared-map
      [overlays]="product | shakemapOverlays:'mmi'"
  ...
```

File: src/app/shakemap/shakemap/shakemap.component.html [hunk 2]
```diff
@@ -8,6 +8,15 @@ <h3>ShakeMap</h3>
 
   <!-- routes/components in ../origin-routing.module.ts -->
   <nav mat-tab-nav-bar>
+    <a mat-tab-link [routerLink]="'./map'"
+        queryParamsHandling="preserve"
+        routerLinkActive #map="routerLinkActive"
+        [queryParams]="{
+          'shakemap-intensity': true
```
Comment (jmfee-usgs) [inline src/app/shakemap/shakemap/shakemap.component.html]: you should not pass this parameter to the shakemap map; only to the interactive map when linking from the static shakemap map component.

File: src/app/shared/map/map.component.ts [hunk 3]
```diff
@@ -177,6 +177,13 @@ export class MapComponent implements AfterViewInit, OnInit {
     return this._showScaleControl;
   }
 
+  fitBounds(bounds) {
```
Comment (jmfee-usgs) [inline src/app/shared/map/map.component.ts]: this should not be a separate method, do this stuff in a setTimeout at the end of the setBounds method.

**Comment by dslosky-usgs** [comment]
Simplified ShakemapOverlaysPipe; optionally pass in an overlay ID to enable it. No longer using logic from InteractiveMapOverlaysPipe

Removed ShakemapMapComponent; using the shared-map instead

**Pull request merged**